flow/activities/flowable.go

```diff
@@ -351,6 +364,13 @@ func (a *FlowableActivity) StartNormalize(
 		return nil, err
 	}
 
+	throughput, err := a.CatalogMirrorMonitor.GetThroughputForCDCBatch(ctx, input.FlowConnectionConfigs.FlowJobName,
```

Comment:
Why are we doing this twice?

Reply:
the first occurrence is the case where a connector doesn't support normalize, we need to log metrics despite not running normalize